### ISSUE TITLE
yt-binlog-groupby without --sort is incorrect

### ISSUE THREAD

**Issue by yoku0825** [yoku0825/ytkit#60]
I cannnot recognize this issue because https://github.com/yoku0825/ytkit/issues/59

```
$ cat t/data/mysqlbinlog_rbr_80.txt | bin/yt-binlog-groupby --group-by=all --cell=1s --sort
180427 11:34:59 d1.t1   INSERT  1
180427 11:35:06 d1.t1   INSERT  1
180427 11:35:19 d1.t1   INSERT  1

$ cat t/data/mysqlbinlog_rbr_80.txt | bin/yt-binlog-groupby --group-by=all --cell=1s
180427 11:34:59 d1.t1   INSERT  1
```

**Comment by yoku0825** [comment]
Even with `--sort` result is incorrect.
Truly, INSERT at 11:34:59 is 2 rows inserted.

```
$ cat t/data/mysqlbinlog_rbr_80.txt | bin/yt-binlog-groupby --group-by=all --cell=1s --sort
180427 11:34:59 d1.t1   INSERT  2
180427 11:35:06 d1.t1   INSERT  1
180427 11:35:19 d1.t1   INSERT  1
```

This is incompatible change.
- Before: more than 1 row in 1 trx, count as 1.
- After: count all rows in 1 trx.

**Comment by yoku0825** [comment]
Fixed by 248a245982dae1d28487c39419c72c3a7b962f57
